src/mumble/ACLEditor.cpp

```diff
@@ -115,6 +115,8 @@ ACLEditor::ACLEditor(int channelid, const MumbleProto::ACL &mea, QWidget *p) : Q
 		QString name = ChanACL::permName(perm);
 
 		if (! name.isEmpty()) {
+			// If the servers version is less than 1.4.0 then it won't support the new permissions to reset a comment/avatar, skipping these iterations of the loop prevents checkboxes for them being added.
+			if ((g.sh->uiVersion < 0x010400) && ((perm == ChanACL::ResetComment) || (perm == ChanACL::ResetTexture))) continue;
```

Comment:
```suggestion
			if ((g.sh->uiVersion < 0x010400) && ((perm == ChanACL::ResetComment) || (perm == ChanACL::ResetTexture))) continue;

```
adding a newline here to indicate that the `if` doesn't belong to the code that's following